src/lightning_app/cli/lightning_cli_create.py

```diff
@@ -54,9 +54,8 @@ def create() -> None:
     "do_async",
     type=bool,
     required=False,
-    default=False,
-    is_flag=True,
-    help="This flag makes the CLI return immediately and lets the cluster creation happen in the background.",
+    default=True,
+    help="This flag makes the CLI wait until cluster creation completes.",
```

Comment:
The help mentions that it's a flag, but it can't be a flag anymore unless we introduce the opposite behavior.

Reply:
:+1: